src/app/interfaces/strongRoute.ts

```diff
@@ -111,7 +111,7 @@ export class StrongRoute {
       throw new Error(msg);
     }
 
-    const prepareParam = (x: StrongRoute) => {
+    const prepareArgs = (x: StrongRoute) => {
```

Comment:
i'd suggest reverting this.

Parameters in routes are known as _Route paramters_ or _query string parameters_. Never arguments?